File: runner.rb
require_relative './classes/school.rb' 

school = School.new('Ridgemont High', '1212 Main st.')

mode = nil 
until mode == '5'
    puts "\nWhat would you like to do?\nOptions:\n1. List All Students\n2. View Individual Student <student_id>\n3. Add a Student\n4. Remove a Student <student_id>\n5. Quit"

    mode = gets.chomp
    puts '' 
    case mode 
    when '1'
        school.list_students
    when '2'
        puts 'Enter student id:'
        puts school.find_student_by_id(gets.chomp)          
    when '5' 
        return 
    end  
end 

File: runner.py
from classes.school import School

school = School('Ridgemont High')

while True:
    mode = input("\nWhat would you like to do?\nOptions:\n1. List All Students\n2. View Individual Student <student_id>\n3. Add a Student\n4. Remove a Student <student_id>\n5. Quit\n")

    if mode == '1':
        school.list_students()
    elif mode == '2':
        student_id = input('Enter student id:')
        student = school.find_student_by_id(student_id)
        print(str(student))
    elif mode == '3':
        student_data = {'role': 'student'}
        student_data['name'] = input('Enter student name:\n')
        student_data['age'] = input('Enter student age: \n')
        student_data['school_id'] = input('Enter student school id: \n')
        student_data['password'] = input('Enter student password: \n')

        school.add_student(student_data)
    elif mode == '4':
        student_id = input('Enter student id:')
        school.remove_student(student_id)
    elif mode == '5':
        break
